### PR TITLE
Chore/switch to jest/grape 14974

### PR DESCRIPTION
**Pull request by nikgraf**
https://jira.ubergrape.com/browse/GRAPE-14974

### CODE REVIEW

File: jest-mocks/globalMocks.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+global.__STATIC_PATH__ = "https://uebergrape.staging.chatgrape.com/"
+global.__DEV__ = true
```
Comment (AleshaOleg) [inline jest-mocks/globalMocks.js]: we don't need `__TEST__` anymore?

**Comment by AleshaOleg** [comment]
@nikgraf Can we also delete `karma.conf.js`?

File: jest-mocks/globalMocks.js (hunk 1 above)
Comment (nikgraf) [reply jest-mocks/globalMocks.js]: It's never used so far. I only wanted to add things that we really need for now.

**Comment by nikgraf** [comment]
@AleshaOleg good catch, just removed the karma conf

**Pull request merged**